spec/e2e/query_proxy_methods_spec.rb

```diff
@@ -511,6 +511,26 @@ def destroy_called
       end
     end
 
+    describe 'as' do
+      it 'changes query variable' do
+        query = Student.as(:stud)
+        expect(query.to_cypher).to include('(stud:`Student`)')
+        query.to_a
+      end
+
+      it 'keeps the current context' do
+        query = Student.where(name: 'John').as(:stud)
+        expect(query.to_cypher).to include('(stud:`Student`)', ' WHERE ')
+        query.to_a
+      end
+
+      it 'keeps the current context with associations' do
+        query = Student.all.lessons.as(:less)
+        expect(query.to_cypher).to include('`Student`)', '(less:`Lesson`)')
```

Comment:
Is passing multiple arguments to `include` an RSpec thing?  Does it mean that we expect both of those strings to be included (like an `and`)?